support-frontend/assets/pages/supporter-plus-thank-you/supporterPlusThankYou.tsx

```diff
@@ -187,7 +190,7 @@ export function SupporterPlusThankYou(): JSX.Element {
 			contributionType === 'ONE_OFF' && email.length > 0,
 			'supportReminder',
 		),
-		// 'feedback', // <-- Temporarily disable supporter plus thank you page survey
+		...maybeThankYouModule(email.length > 0, 'feedback'),
```

Comment:
Do we sometimes not have an email address at this point?

Reply:
I don't know of a production scenario where you would get through the checkout to the thank-you without providing an email address (don't you need this for all checkouts?). Is there another route through I have not spotted?

Reply:
Not that I know of, just wondering why you have the length check on the email address? (Happy to leave it in, but just not sure if it’s necessary)

Reply:
Paul Heron made a good point when checking, it also stops anyone spamming the thankyou url direct . Thus stopping  being able to post multiple survey's without an associated checkout.